manifest.json

```diff
@@ -20,6 +20,7 @@
       "images/logo_48.png",
       "privly-applications/PlainPost/show.html",
       "privly-applications/Message/show.html",
+      "privly-applications/Message/new_embed.html",
```

Comment:
I am pulling this now to start with testing, but I think this is the line we most need to talk about. Currently the show.html application that gets injected is the same one that is viewed when the user clicks through to view it. I think this may be the proper approach for the new.html application as well, but I need to dive into the code more to be certain.